src/App/Pages/Accounts/LoginPage.xaml

```diff
@@ -106,14 +106,21 @@
                                 StyleClass="btn-primary"
                                 Clicked="LogIn_Clicked" />
                         <controls:IconLabelButton
+                            HorizontalOptions="FillAndExpand"
+                            VerticalOptions="CenterAndExpand"
                             Icon="{Binding Source={x:Static core:BitwardenIcons.Device}}"
                             Label="{u:I18n LogInWithAnotherDevice}"
                             ButtonCommand="{Binding LogInCommand}"
                             IsVisible="{Binding IsKnownDevice}"/>
                         <controls:IconLabelButton
+                            HorizontalOptions="FillAndExpand"
```

Comment:
This can be removed, the `StackLayout` is vertical so expanding horizontally does nothing